eng/pipelines/templates/stages/archetype-sdk-tool-dotnet.yml

```diff
@@ -85,9 +91,29 @@ stages:
               testRunTitle: 'Windows DotNet $(DotNetCoreVersion)'
               testResultsFormat: 'VSTest'
               mergeTestResults: true
-
-  - ${{if and(ne(variables['Build.Reason'], 'PullRequest'), eq(variables['System.TeamProject'], 'internal'))}}:
-    - template: pipelines/stages/net-release-to-feed.yml@azure-sdk-build-tools
-      parameters:
-        # Publish to https://dev.azure.com/azure-sdk/public/_packaging?_a=feed&feed=azure-sdk-for-net
-        DevOpsFeedId: '29ec6040-b234-4e31-b139-33dc4287b756/fa8c16a3-dbe0-4de2-a297-03065ec1ba3f'
+  
+  - ${{if and(eq(variables['Build.Reason'], 'Manual'), eq(variables['System.TeamProject'], 'internal'))}}:
+    - stage: Release
+      jobs:
+      - deployment: Sign_And_Publish
+        environment: net-release
+        pool:
+          vmImage: windows-2019
+      
+        strategy:
+          runOnce:
+            deploy:
+              steps:
+                - checkout: azure-sdk-for-net
```

Comment:
I think if we do sparse checkout here it will save a LOT of time, given that we only need to clone the eng directory.

Reply:
The sparse checkout template does not seem to provide a way to checkout a specific branch. i.e without having to use the commit SHA.

Reply:
You can pass in a branch name via the `commitish` parameter (a git commitish can be a SHA, a refname/branch name, a revision, etc.).

Reply:
Thanks a lot. I didn't know that.